uicomponent-compose/other/src/main/java/io/github/droidkaigi/feeder/other/OtherScreen.kt

```diff
@@ -185,7 +178,10 @@ private fun AppBar(
 fun PreviewOtherScreen() {
     ConferenceAppFeederTheme {
         OtherScreen(
-            initialSelectedTab = OtherTabs.AboutThisApp,
+            selectedTab = OtherTabs.AboutThisApp,
+            onSelectTab = {
+
```

Comment:
Could you remove this empty line, please?

Reply:
ok! thankyou😆